doc/rose-rug-suites-I.html

```diff
@@ -633,51 +633,57 @@ <h3 class="alwayshidden">Jinja2 rose-suite.conf Metadata Example</h3>
     </div>
 
     <div class="slide">
+      <h2 id="utc-mode">UTC mode</h2>
+
+      <p>We recommend running all suites in UTC mode. E.g.:</p>
+      <pre class="prettyprint lang-cylc">
+[cylc]
+    UTC mode = True # Ignore DST
+</pre>
+    </div>
+
+    <div class="slide">
       <h2 id="event-hooks">Event Hooks</h2>
 
-      <p>Suites also need some standard Rose <em>event hooks</em> to construct
-      the suite log viewer (more later) and report or shutdown on failure.</p>
+      <p>Suites can have event handlers to report events or shutdown on
+      failure.</p>
 
-      <p><strong>The following should be in every Rose suite:</strong></p>
       <pre class="prettyprint lang-cylc">
 [cylc]
-    UTC mode = True # Ignore DST
+    # abort if any task fails = True
     [[event hooks]]
-        timeout handler = rose suite-hook --mail --shutdown
-        timeout = P3D
+        # timeout handler = rose suite-hook --mail --shutdown
+        # timeout = P3D
 </pre>
     </div>
 
     <div class="slide">
       <h3 class="alwayshidden">Event Hooks Explained</h3>
 
-      <p>Here, when the time since the last task state change exceeds the value
-      of <kbd>timeout</kbd> the suite times out, the user will receive an email
-      (<kbd>--mail</kbd>) and the suite will shutdown
-      (<kbd>--shutdown</kbd>).</p>
+      <p>Normally, when a task fails, the suite will continue to run as much as
+      possible and wait for user input to fix (and perhaps retry) the failure so
+      it can continue. If <code>abort if any task fails = True</code> is not
+      commented out, the suite will abort as soon as a task fails.</p>
     </div>
 
     <div class="slide">
       <h3 class="alwayshidden">Hello World Event Hooks (Continued)</h3>
 
-      <p>Suites should also have task event hooks, along these lines:</p>
+      <p>Suites can also be configured to email you on specific events. For
+      example:</p>
       <pre class="prettyprint lang-cylc">
 [runtime]
     [[root]]
-        [[[event hooks]]]
-           succeeded handler = rose suite-hook
-           failed handler = rose suite-hook --shutdown
-           retry handler = rose suite-hook --mail
-           submission failed handler = rose suite-hook --shutdown
-           submission timeout handler = rose suite-hook --mail
-           execution timeout handler = rose suite-hook --mail
-           submission timeout = P1D
-           execution timeout = PT3H
+        [[[events]]]
+            mail events = submission retry, submission failed, submission timeout, \
+                          retry, failed, execution timeout
+            # submission timeout = PT3H
```

Comment:
In which case I'd say either remove them completely or keep them in the suite uncommented - the documentation should cover any site running Rose not just here and therefore can't make assumptions about what local settings may or may not be (and as its a cylc setting we can't recommend what that default should be in the Rose part of the documentation).